src/components/Favorite.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react'
+import styled from '@emotion/styled'
+import Image from './Image'
+import Heart from './Heart'
+
+interface FavoriteProps {
+  favoriteList: string[],
+  unfavorite(url: string): any,
+}
+
+const Favorite: React.SFC<FavoriteProps> = (props) => {
+  return props.favoriteList == undefined ? null : generateImages(props.unfavorite, props.favoriteList);
+}
+
+function generateImages(unfavorite, favoriteList) {
+  const images = []
+  favoriteList.forEach((element, idx) => {
+    images.push(
+      <Image 
+        url={element} 
+        width={128} 
+        height={128} 
+        onClick={unfavorite}
+        favorited={true}
+        fromSearch={false}
+        key={idx}
+      />);
```

Comment:
Is there a more succinct Array method available here? Could we use `.map()` instead?